lib/engine/part/city.rb

```diff
@@ -49,6 +49,10 @@ def add_reservation!(entity, slot = nil)
         end
       end
 
+      def remove_reservation!(entity)
```

Comment:
unecessary since resarvations is public

can just do action.city.reservations.delete in specialnwr unless you play to do other stuff in this helper